src/controller/sms/get_step_name.js

```diff
@@ -0,0 +1,18 @@
+const steps = require('./steps')
+const debug = require('debug')('sms') // eslint-disable-line
+
+/**
+ * getNextStepName - gets the current step name based on session data
+ *
+ * @param  {object} session session object from request
+ * @return {string} name of current step
+ */
+module.exports = function getStepName(session) {
+  const stepName = Object.keys(steps).reduce((memo, step) => {
+    if (memo) return memo
+    if (!session[step]) return step
```

Comment:
I'm a bit out of my league, but am I reading right that at this point we're storing things directly on the session using the step name? This seems a bit fragile--if I'm right and that's what we're doing, what do you think about storing it under a "steps" key or something? I don't know if that makes it all of a sudden crazy difficult or confusing.